infra/feast-operator/test/utils/test_util.go

```diff
@@ -446,3 +449,21 @@ func GetSimplePreviousVerCR() string {
 func GetRemoteRegistryPreviousVerCR() string {
 	return fmt.Sprintf("https://raw.githubusercontent.com/feast-dev/feast/refs/tags/v%s/infra/feast-operator/test/testdata/feast_integration_test_crs/v1alpha1_remote_registry_featurestore.yaml", feastversion.FeastVersion)
 }
+
+func CreateNamespace(namespace string, testDir string) error {
+	cmd := exec.Command("kubectl", "create", "ns", namespace)
+	output, err := Run(cmd, testDir)
+	if err != nil {
+		return fmt.Errorf("failed to create namespace %s: %v\nOutput: %s", namespace, err, output)
+	}
+	return nil
+}
+
+func DeleteNamespace(namespace string, testDir string) error {
+	cmd := exec.Command("kubectl", "delete", "ns", namespace, "--wait=true")
```

Comment:
Can we have some timeout here fo this command execution?

Reply:
sure updated timeout to 3min `--timeout=180s`